opensrp-reveal/src/main/res/values-th/strings.xml

```diff
@@ -128,6 +128,10 @@
     <string name="mosquito_collection_trap_set_date">กำหนดวันที่:&#160;</string>
     <string name="mosquito_collection_trap_follow_up_date">วันที่ติดตาม:&#160;</string>
 
+    <string name="paot_last_updated_date_test_text">Last Updated Date:&#160;</string>
```

Comment:
Use the Thai labels on 
https://docs.google.com/spreadsheets/d/1kGPgyF0eJdag19FFL1Oog2gdphdPtKobY4EQ0gMgtgw/edit#gid=227143838